packages/es-components/src/components/controls/radio-buttons/RadioButton.js

```diff
@@ -130,7 +138,8 @@ RadioButton.propTypes = {
 
 RadioButton.defaultProps = {
   inline: true,
-  validationState: 'default'
+  validationState: 'default',
+  descriptorIds: ''
```

Comment:
Could you update a radio button example to use the new props so it's easier to understand what they're used for?